src/themes/elife/styles.css

```
@@ -8,7 +8,7 @@
 /* N.B. Until eLife's pattern library CSS is available from an npm module, this URL
    needs to be updated every time eLife deploy CSS.
    Once the CSS is available from an npm module, update this to @import accordingly and delete this comment :-) */
-@import 'https://elifesciences.org/assets/patterns/css/all.77dea4bb.css';
+@import '../../../node_modules/@elifesciences/pattern-library/source/assets/css/all.css';
```

Could you please change the import path to avoid dependency on the folder structure please?
```suggestion
@import '@elifesciences/pattern-library/source/assets/css/all.css';
```

---

As an extra step, if you wanted to abstract away the path and declare this file as the main CSS entry point, you could add a [`style` field](https://github.com/twbs/bootstrap/blob/master/package.json#L74) to the `@elifesciences/pattern-library/package.json`.
This would allow you to make changes to the project structure and file names in the future. Purely optional, but thought it would be something you might want to consider before merging this PR.

So in [`package.json`](https://github.com/elifesciences/pattern-library/blob/develop/package.json):

``` diff
{
  // …
+  "style": "source/assets/css/all.css"
  // …
}
```

and then import would become:

``` css
@import '@elifesciences/pattern-library';
```